pkg/command/config.go

```diff
@@ -243,8 +243,9 @@ var initConfigCmd = &cobra.Command{
 
 // Note: Shall be deprecated in a future version. Superseded by 'tanzu context' command.
 var serversCmd = &cobra.Command{
-	Use:   "server",
-	Short: "Configured servers",
+	Use:        "server",
+	Short:      "Configured servers",
+	Deprecated: fmt.Sprintf("will be removed in version %q. Use the %q command instead.", "1.2.0", "context"),
```

Comment:
This is fine, however, it would be nice if we can use https://github.com/vmware-tanzu/tanzu-plugin-runtime/blob/main/command/deprecation.go#L20 library to be consistent.

Reply:
Good point, I wasn't aware if this library.
I have updated both my change and the deprecation of the "login" command to use this lib.